backend/runner/schemasync/syncer.go

```diff
@@ -326,7 +326,9 @@ func (s *Syncer) SyncInstance(ctx context.Context, instance *store.InstanceMessa
 			if err != nil {
 				return nil, nil, nil, errors.Wrapf(err, "failed to create instance %q database %q in sync runner", instance.ResourceID, databaseMetadata.Name)
 			}
-			newDatabases = append(newDatabases, newDatabase)
+			if newDatabase != nil {
```

Comment:
Anywhere that could cause newDatabase to be nil?

Reply:
Still investigating.. It's really strange for the nil pointer panic..

Reply:
Thanks. Happy about merging this PR to stop the bleeding. Just curious what could happen.